ldes-server-port-fetch-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/eventstream/EventStreamController.java

```diff
@@ -32,7 +36,13 @@ public EventStreamController(RestConfig restConfig, CachingStrategy cachingStrat
 	@CrossOrigin(origins = "*", allowedHeaders = "")
 	@GetMapping(value = "{collectionname}")
 	@Operation(summary = "Retrieve an Linked Data Event Stream")
-	public ResponseEntity<EventStreamResponse> retrieveLdesFragment(@RequestHeader(HttpHeaders.ACCEPT) String language,
+	@ApiResponse(responseCode = "200", content = {
+			@Content(mediaType = contentTypeNQuads),
+			@Content(mediaType = contentTypeJSONLD),
+			@Content(mediaType = contentTypeTurtle)
+	})
+	public ResponseEntity<EventStreamResponse> retrieveLdesFragment(
+			@Parameter(hidden = true) @RequestHeader(HttpHeaders.ACCEPT) String language,
```

Comment:
Minor question on this: Should this (Parameter(hidden = true)) also be added in the TreeNodeController?